src/weatherreport/DEVELOPMENT.md

```diff
@@ -0,0 +1,73 @@
+# WeatherReport Development
```

Comment:
If we're going to make this part of CouchDB proper, a) this file needs to go, b) this needs to be built as part of the regular top-level `make` process, and c) `make test` for top-level CouchDB should include a target that starts up a cluster with `dev/run` and runs `weatherreport` against it. The test should verify by checking for a zero exit status code from `weatherreport`.

It looks like the local `Makefile` just does the usual `rebar` things, so perhaps this entire file (and the `Makefile`) can simply be dropped.

Reply:
I updated the CouchDB Makefile with several improvements, including a `weatherreport-test` target that passes, and which is also run in the `check` target.

This repo's Makefile was also removed.